pubsub/spec/topics_spec.rb

```diff
@@ -177,6 +177,7 @@ def expect_with_retry attempts: 5
   end
 
   it "publishes messages with batch settings" do
+    sleep(30)
```

Comment:
What if we add a `before :each` with this sleep instead?

Reply:
Sure :)

Reply:
Hi Frank! I am thinking if we can just set `config.default_sleep_interval` to `40` instead of `10`; would it be better than adding `sleep(30)` to `before :each`?